website/content/docs/connect/config-entries/ingress-gateway.mdx

```diff
@@ -1076,6 +1076,32 @@ You can specify the following parameters to configure ingress gateway configurat
               respected, a L7 protocol must be defined in the \`protocol\` field.
               If not specified, it uses the default value. For example, 1024 for Envoy proxy.`,
         },
+        {
+          name: 'PassiveHealthCheck',
+          type: 'PassiveHealthCheck: <optional>',
+          description:
+            'Passive health checks are used to remove hosts from the upstream cluster which are unreachable or are returning errors.',
+          children: [
+            {
+              name: 'interval',
+              type: 'int: <optional>',
+              description:
+                "The time in nanosecond between checks. Each check will cause hosts which have exceeded `max_failures` to be removed from the load balancer, and any hosts which have passed their ejection time to be returned to the load balancer. If not specified, it uses the default value. For example, 10s for Envoy proxy.",
```

Comment:
```suggestion
                "The time in nanosecond between checks. If the number of failed checks exceed the `max_failures`, Consul removes the failing host from the load balancer. Additionally, hosts that have passed their ejection time are returned to the load balancer. If not specified, Consul uses the proxy's default value. For example, `10s` for Envoy proxy.",
```
Is that clearer without changing the meaning? Also, what is the "ejection time"? I didn't see that phrase anywhere else on the page.

Reply:
@trujillo-adam , thanks for helping editing the text. Actually I copied these sentences from [other places](https://developer.hashicorp.com/consul/docs/connect/proxies/envoy#passive_health_check) where passive health check is enabled. I like your edited version, so will overwrite the other place in the following commit.